yt/frontends/ytdata/data_structures.py

```diff
@@ -625,7 +624,7 @@ def get_data(self, fields=None):
         if len(fields_to_get) == 0 and len(fields_to_generate) == 0:
             return
         elif self._locked:
-            raise GenerationInProgress(fields)
+            raise YTGenerationInProgress(fields)
```

Comment:
Couldn't figure out where ` GenerationInProgress` was supposed to be defined, so isn't this line a bugfix ?

Reply:
Well, sorta.  I actually renamed *and* moved it.  It used to just be `GenerationInProgress` but now I moved it to `yt/utilities/exceptions.py` and renamed it with the prefix.

Reply:
Any chance we'd except some downstream code to rely on catching this ? If so, maybe we should apply a similar pattern as here https://github.com/yt-project/yt/pull/2818